tests/operator_tests.py

```diff
@@ -409,7 +409,7 @@ def test_concatenate_type_preservation(self):
         with self.assertRaises(TypeError):
             ak.concatenate(datetime_one, bitvector_two)
 
-        # verify ak.util.concatenate still works
```

Comment:
Again, I believe this is actually referring to concatenate in `arkouda/util`

Reply:
Fixed